libs/dh/admin/data-access-api/src/lib/dh-admin-edit-user.store.ts

```diff
@@ -36,11 +43,11 @@ const initialState: State = {
 
 @Injectable()
 export class DhAdminEditUserStore extends ComponentStore<State> {
-  private readonly userRolesStore = inject(DhAdminUserRolesStore);
+  private readonly apollo = inject(Apollo);
```

Comment:
question: Have you tried with the new `mutation` utility? I'm just interested to know how or if it works with stores. Not that you have to rewrite everything again just for that 🙂

Reply:
I had already written the mutation before your util were merged in :-)